Tests/SecureEnclaveTests.swift

```diff
@@ -126,10 +127,12 @@ class SecureEnclaveTests: XCTestCase
         let sharedAccessGroupIdentifier: Identifier
         #if os(iOS)
             sharedAccessGroupIdentifier = Identifier(nonEmpty: "com.squareup.Valet-iOS-Test-Host-App")!
-        #elseif os(OSX)
+        #elseif os(macOS)
             sharedAccessGroupIdentifier = Identifier(nonEmpty: "com.squareup.Valet-macOS-Test-Host-App")!
         #elseif os(tvOS)
             sharedAccessGroupIdentifier = Identifier(nonEmpty: "com.squareup.Valet-tvOS-Test-Host-App")!
+        #elseif os(watchOS)
+            sharedAccessGroupIdentifier = Identifier(nonEmpty: "com.squareup.ValetTouchIDTestApp.watchkitapp.watchkitextension")!
```

Comment:
Nit: Can we standardize this name like the others?

Reply:
solid catch!

Reply:
Okay, so, after a small investigation this is a rabbit hole I'd prefer to avoid (or fix in a later PR). I tied the watch test to the `ValetTouchIDTestApp`, and... the way WatchKit apps/extensions work every extension needs to share the same prefix. So unless I want to change _all_ the bundle IDs involved here... this is probably better worth skipping for now.

Reply:
👍 seems reasonable